EFAuditer/EFAuditer.csproj

```diff
@@ -7,6 +7,7 @@
 
   <ItemGroup>
     <PackageReference Include="Audit.EntityFramework.Core" Version="17.0.4" />
+    <PackageReference Include="Lindhart.Analyser.MissingAwaitWarning" Version="2.0.0" />

```

Comment:
The selection of projects for which we've included this package seems strange. Shouldn't it be included for all of them?

Reply:
I went for all the projects with code that we "run" but now that you point it out we might as well have them for all of them. I'll add those in before I merge